giraffe/README.md

```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
+
+   ```html
+   <!DOCTYPE html>
+   <html lang="en">
+     <head>
+       <meta charset="utf-8" />
+       <title>Giraffe</title>
+     </head>
+
+     <body id="home">
+       <main id="root"></main>
+     </body>
+   </html>
+   ```
+
+   Save this file as `index.html`.
 
 1. Add script imports of React, React-DOM, and Giraffe to the bottom of the `head` section.
-    ```html
-      <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
-      <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
-      <script crossorigin src="https://unpkg.com/@influxdata/giraffe"></script>
-    </head>
-    ```
+   ```html
+     <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
+     <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+     <script crossorigin src="https://unpkg.com/@influxdata/giraffe"></script>
+   </head>
+   ```
 1. Wire up a simple React element. Add a script tag under `main` in the `body` of the html document, and then render a simple element onto the page using React.
-    ```html
-      <main id="root"></main>
-      <script type="text/javascript">
-        ReactDOM.render(
-          React.createElement('h1', null, 'Hello World'),
-          document.getElementById('root')
-        );
-      </script>
-    </body>
-    ```
+
+   ```html
+     <main id="root"></main>
+     <script type="text/javascript">
+       ReactDOM.render(
+         React.createElement('h1', null, 'Hello World'),
+         document.getElementById('root')
+       );
+     </script>
+   </body>
+   ```
 
 1. Open `index.html` in a browser. You should see a header with Hello World as the text.
 
 1. Create a wrapper element for the Giraffe `Plot` we'll be rendering into. Add it to the script tag under `root`:
-    ```html
-    <main id="root"></main>
-      <script type="text/javascript">
-        class PlotRenderer extends React.Component {
-          render() {
-            const style = {
-              width: "calc(70vw - 20px)",
-              height: "calc(70vh - 20px)",
-              margin: "40px",
-            };
-            return React.createElement('div', {style}, 'Giraffe Plot Goes Here');
-          }
-        }
-      <script>
-    ```
+
+   ```html
+   <main id="root"></main>
+     <script type="text/javascript">
+       class PlotRenderer extends React.Component {
+         render() {
+           const style = {
+             width: "calc(70vw - 20px)",
+             height: "calc(70vh - 20px)",
+             margin: "40px",
+           };
+           return React.createElement('div', {style}, 'Giraffe Plot Goes Here');
+         }
+       }
+     <script>
+   ```
 
 1. And have React render that element. Change the `ReactDOM.render` call to:
-    ```js
-    ReactDOM.render(
-      React.createElement(PlotRenderer, null, null),
-      document.getElementById('root')
-    );
```

Comment:
i'd prefer if you didn't remove semicolons - those were put in deliberately. i know we don't use them in our conventions, but they're required in JavaScript expressions, and in an html quickstart that's copypasta-ed into an indexl.html file, they won't be built and inserted into the output. which means we'd be at the mercy of the compiler guessing our intentions.